app/scripts/views/base.js

```diff
@@ -762,6 +762,13 @@ define(function (require, exports, module) {
       this._hasNavigated = true;
     },
 
+    navigateViaServer (url) {
```

Comment:
Can you add a jsdoc here?  

The method name, while describing exactly what it does, seems strange. `serverNavigate` seems equally strange. `navigate(url, null, { server: true })` seems verbose.

What if it's something a bit more generic that can be used elsewhere, something like `navigateToUrl`. The name implies it's a hard redirect instead of via pushState. In particular, we could that to [replace this bit of code in the external-links-mixin](https://github.com/mozilla/fxa-content-server/blob/9950eb4aafc98c0716bb029bad58a39aa3212aa2/app/scripts/views/mixins/external-links-mixin.js#L126:L133).